VamprTests/5_ClosestCommonAncestorTests.swift

```diff
@@ -27,14 +27,16 @@ class ClosestCommonAncestorTests: XCTestCase {
     super.setUp()
     
     rootVampire = Vampire(name: "original", yearConverted: 0)
-    offspring.0 = Vampire(name: "", yearConverted: 0)
-    offspring.1 = Vampire(name: "", yearConverted: 0)
-    offspring.2 = Vampire(name: "", yearConverted: 0)
-    offspring.3 = Vampire(name: "", yearConverted: 0)
-    offspring.4 = Vampire(name: "", yearConverted: 0)
-    offspring.5 = Vampire(name: "", yearConverted: 0)
-    offspring.6 = Vampire(name: "", yearConverted: 0)
-    offspring.7 = Vampire(name: "", yearConverted: 0)
+    let v0 = Vampire(name: "", yearConverted: 0)
+    let v1 = Vampire(name: "", yearConverted: 0)
+    let v2 = Vampire(name: "", yearConverted: 0)
+    let v3 = Vampire(name: "", yearConverted: 0)
+    let v4 = Vampire(name: "", yearConverted: 0)
+    let v5 = Vampire(name: "", yearConverted: 0)
+    let v6 = Vampire(name: "", yearConverted: 0)
+    let v7 = Vampire(name: "", yearConverted: 0)
+    
+    offspring = (v0, v1, v2, v3, v4, v5, v6, v7)
```

Comment:
@thomasMinshull What's the improvement to do it this way?

Reply:
It runs :P